signal-hook-registry/src/lib.rs

```diff
@@ -435,23 +554,57 @@ mod tests {
     use std::thread;
     use std::time::Duration;
 
+    #[cfg(not(windows))]
     use libc::{pid_t, SIGUSR1, SIGUSR2};
 
+    #[cfg(windows)]
+    use libc::SIGTERM as SIGUSR1;
+    #[cfg(windows)]
+    use libc::SIGTERM as SIGUSR2;
```

Comment:
:thinking: I wonder why the tests used separate signals. Could something bad happen? One test sending a signal to another? Invoke the race condition? Because tests are allowed to run in parallel.

I don't have a solution ready and this is inside tests only, so I'd propose to simply add a FIXME note warning about it; if the tests fail because of the race condition, such note will help understand what happened.

Reply:
You mean to use `SIGUSR1` in both cases (in the tests for POSIX)?

Reply:
No. I'm just worrying that with SIGUSR1 and SIGUSR2 being the same signal in windows tests (as you define them), one test might somehow (not sure how) could make the other test fail by accident.

I'm proposing to keep the code as it is, just add a note with this worry, so *if* the problem ever happens, people will just try re-running it and not invest hours trying to figure out what will have happened.